src/ConnectedMode/Binding/IUnintrusiveBindingController.cs

```diff
@@ -67,6 +70,17 @@ public async Task BindAsync(BoundServerProject project, IProgress<FixedStepsProg
             await bindingProcess.SaveServerExclusionsAsync(token);
         }
 
+        public bool Unbind(string localBindingKey)
```

Comment:
this could be a separate PR, but you can leave it as-is

Reply:
Maybe, considering a potential change to ASBT, it could actually deserve its own PR